### ISSUE TITLE
test.py in 1.x branch is much slower than in 0.x branch

### ISSUE THREAD

**Issue by liuheng92** [open-mmlab/mmocr#1470]
### 1.x log（partial）
10/19 09:21:43 - mmengine - INFO - Config:
file_client_args = dict(backend='disk')
test_pipeline = [
    dict(
        type='LoadImageFromFile',
        file_client_args=dict(backend='disk'),
        color_type='color_ignore_orientation'),
    dict(type='Resize', scale=(2944, 640), keep_ratio=True),
    dict(
        type='LoadOCRAnnotations',
        with_polygon=True,
        with_bbox=True,
        with_label=True),
    dict(type='Pad', size_divisor=32),
    dict(
        type='PackTextDetInputs',
        meta_keys=('img_path', 'ori_shape', 'img_shape', 'scale_factor'))
]
std_data_root = './data/total/'
default_scope = 'mmocr'
log_level = 'INFO'
test_dataloader = dict(
    batch_size=1,
    num_workers=4,
    persistent_workers=True,
    sampler=dict(type='DefaultSampler', shuffle=False),
    dataset=dict(
        type='OCRDataset',
        data_root='./data/total/',
        ann_file='instances_val.json',
        data_prefix=dict(img_path=''),
        test_mode=True,
        pipeline=[
            dict(
                type='LoadImageFromFile',
                file_client_args=dict(backend='disk'),
                color_type='color_ignore_orientation'),
            dict(type='Resize', scale=(2944, 640), keep_ratio=True),
            dict(
                type='LoadOCRAnnotations',
                with_polygon=True,
                with_bbox=True,
                with_label=True),
            dict(type='Pad', size_divisor=32),
            dict(
                type='PackTextDetInputs',
                meta_keys=('img_path', 'ori_shape', 'img_shape',
                           'scale_factor'))
        ]))
launcher = 'none'
work_dir = './work_dirs/'

Result has been saved to /mmocr1.x/work_dirs/modules_statistic_results.json
10/19 09:21:50 - mmengine - INFO - Distributed training is not used, all SyncBatchNorm (SyncBN) layers in the model will be automatically reverted to BatchNormXd layers if they are used.
local loads checkpoint from path: ./work_dirs/model/epoch_440.pth
10/19 09:21:53 - mmengine - INFO - Load checkpoint from ./work_dirs/model/epoch_440.pth
10/19 09:21:53 - mmengine - INFO - Epoch(test) [5/2811]    memory: 2209  
10/19 09:21:55 - mmengine - INFO - Epoch(test) [10/2811]    eta: 0:21:15  time: 0.4553  data_time: 0.0381  memory: 2227  
10/19 09:21:56 - mmengine - INFO - Epoch(test) [15/2811]    eta: 0:21:15  time: 0.3192  data_time: 0.0040  memory: 2217  
10/19 09:21:58 - mmengine - INFO - Epoch(test) [20/2811]    eta: 0:15:08  time: 0.3255  data_time: 0.0054  memory: 2220  
...........
0/19 09:30:02 - mmengine - INFO - Epoch(test) [2780/2811]    eta: 0:00:05  time: 0.1836  data_time: 0.0081  memory: 185  
10/19 09:30:03 - mmengine - INFO - Epoch(test) [2785/2811]    eta: 0:00:05  time: 0.1711  data_time: 0.0092  memory: 307  
10/19 09:30:04 - mmengine - INFO - Epoch(test) [2790/2811]    eta: 0:00:03  time: 0.1570  data_time: 0.0129  memory: 307  
10/19 09:30:05 - mmengine - INFO - Epoch(test) [2795/2811]    eta: 0:00:03  time: 0.2197  data_time: 0.0135  memory: 4027  
10/19 09:30:05 - mmengine - INFO - Epoch(test) [2800/2811]    eta: 0:00:02  time: 0.1821  data_time: 0.0116  memory: 282  
10/19 09:30:06 - mmengine - INFO - Epoch(test) [2805/2811]    eta: 0:00:02  time: 0.1389  data_time: 0.0120  memory: 170  
10/19 09:30:07 - mmengine - INFO - Epoch(test) [2810/2811]    eta: 0:00:00  time: 0.1688  data_time: 0.0102  memory: 185  
10/19 09:30:07 - mmengine - INFO - Evaluating hmean-iou...
### 0.x log
[>>>>>>>>>>>>>>>>>>>>>>>>] 2811/2811, 30.5 task/s, elapsed: 92s, ETA:     0smodel duration, total_time:80.08576536178589, avg_time:0.028490133533186014

**Comment by liuheng92** [comment]
This is not an issue, It's about "print logical", like "compute first or print first". So I close this issue now.
